.flowconfig

```diff
@@ -4,6 +4,7 @@
 <PROJECT_ROOT>/src/docs/build/
 .+/.+\.gzip\.js
 .+/\.next/.+
+<PROJECT_ROOT>/src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures
```

Comment:
Would it be possible to make the fixtures valid (real-world examples) to avoid this? 🤔

Reply:
Actually yes. Not outputs but inputs yes.